pkg/types/ccipocr3/common_types.go

```diff
@@ -85,3 +86,29 @@ func (b *BigInt) UnmarshalJSON(p []byte) error {
 func (b BigInt) IsEmpty() bool {
 	return b.Int == nil
 }
+
+type P2PID [32]byte
```

Comment:
There is also a `Bytes32` type

Reply:
I removed this completely and using libocr PeerID 👍